replication/client/client.go

```diff
@@ -332,6 +333,7 @@ func (c *Replicator) Start(progressChan <-chan uint64) {
 				continue
 			}
 
+			fmt.Println("returning")
```

Comment:
update this to a log (or if this is for test, remove this)

Reply:
Oops left that during debugging.